test/transactions/0_send.js

```diff
@@ -145,6 +145,10 @@ describe('#send transaction', () => {
 						.and.be.type('object')
 						.and.be.empty();
 				});
+
+				it('should not have the second signature property', () => {
+					sendTransaction.should.not.have.property('signSignature');
+				});
```

Comment:
In other test files you've moved these tests out of the `returned transaction block`. Best to be consistent.